src/attributes.jl

```diff
@@ -23,6 +23,23 @@ abstract type AbstractConstraintAttribute end
 
 const AnyAttribute = Union{AbstractInstanceAttribute, AbstractVariableAttribute, AbstractConstraintAttribute}
 
+"""
+    supportsattribute(instance::AbstractInstance, attr::AbstractInstanceAttribute, ::Type{V}) where V
+
+Return a `Bool` indicating whether there exists an instance `src` that contains the attribute `attr` and `get(src, attr)` is of type `V` such that `copy!(instance, src)` returns a `CopyResult` object with a status `CopySuccess`.
```

Comment:
s/`src`/`instance`/

Reply:
I think it should be `src`, it is different to the `instance` that is passed as first argument